docs/source/building/windows.rst

```diff
@@ -15,31 +15,26 @@ Install dependencies:
    .. code-block:: bash
 
       pacman -S \
-        base-devel \
-        cmake \
-        diffutils \
-        gcc \
         git \
```

Comment:
```suggestion
        base-devel \
        cmake \
        diffutils \
        gcc \
        git \
        make \
```

Reply:
If `ninja` is absolutely required then add that instead of `make`

Reply:
After building with make, this is no longer required

Reply:
Deps in the docs should closely match the CI.

Reply:
I removed these extraneous dependencies from the CI description.

The build passes: https://github.com/tez011/Sunshine/actions/runs/8034826419/job/21946815487#step:3:5